Cartfile.resolved

```diff
@@ -1,8 +1,8 @@
-binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" "23.4.0"
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" "131.0.0"
+binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" "23.5.0-beta.1"
```

Comment:
I guess it should be 23.5.0-rc.1, right?

Reply:
You are right, missed common line...